### DISCUSSION TITLE
Does Neovide require all configuration to happen in `init.lua`?

### DISCUSSION THREAD

**Discussion by adrian5** [neovide/neovide#2279] [Q&A] [answered]
My nvim config is fairly modular and I'm importing most vim options into `init.lua` through a secondary file `options.lua` (`require("options")`). I noticed that e.g. `listchars` used default values in Neovide unless I set it in `init.lua` itself. Is this a known limitation?

**Comment by fredizzimo** [comment, marked as answer]
No, it loads the init files exacly like regular terminal mode nvim. With the exception that a `ginit.vim` file is loaded before everything else, if present.

I have no idea why it's not working for you, it's working for me and a lot of other people.
